Solutions/LastPass/Data Connectors/LastPassAPIConnector.json

```diff
@@ -0,0 +1,123 @@
+                    "permissions": {
+                        "resourceProvider": [{
+                                "provider": "Microsoft.OperationalInsights/workspaces",
+                                "permissionsDisplayText": "read and write permissions are required.",
+                                "providerDisplayName": "Workspace",
+                                "scope": "Workspace",
+                                "requiredPermissions": {
+                                    "action": true,
+                                    "write": true,
+                                    "read": true,
+                                    "delete": true
+                                }
+                            }
+                        ],
+                        "customs": [ {
+                                "name": "LastPass API Key",
+                                "description": "A LastPass API key is required. [See the documentation to learn more about LastPass API](https://support.logmeininc.com/lastpass/help/use-the-lastpass-provisioning-api-lp010068)."
+                            }
+                        ]
+                    },
+                    "instructionSteps": [
+                        {
+                            "title": "Connect LastPass Enterprise to Azure Sentinel",
+                            "description": "Provide the LastPass Provisioning API Key.",
+                            "instructions": [
+                                {
+                                    "parameters": {
+                                        "enable": "true"
+                                    },
+                                    "type": "APIKey"
+                                }
+                            ]
+                        }
+                    ]
+                },
+                "pollingConfig": 
+                    {
+                        "owner": "ASI",
+                        "version": "2.0",
+                        "source": "PaaS",
+                        "auth": {
+                            "authType": "APIKey",
+                            "APIKeyName": "provhash",
+                            "IsAPIKeyInPostPayload": true
+                        },
+                        "request": {
+                            "apiEndpoint": "https://lastpass.com/enterpriseapi.php",
+                            "rateLimitQPS": 2,
+                            "httpMethod": "Post",
+                            "queryTimeFormat": "yyyy-MM-ddTHH:mm:ssZ",
+                            "retryCount": 3,
+                            "queryWindowInMin": 10,
+                            "timeoutInSeconds": 120,
+                            "queryParametersTemplate": "{'cid': '12537091', 'cmd': 'reporting', 'data': { 'from': '{_QueryWindowStartTime}', 'to': '{_QueryWindowEndTime}' }, '{_APIKeyName}': '{_APIKey}'}",
+                            "isPostPayloadJson": true
+                        },
```

Comment:
Please define anyof attribute and add endTimeAttributeName and startTimeAttributeName property.